SwiftWisdom/Core/Foundation/Time/Date+Extensions.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Date+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Paul Rolfe on 1/28/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+public extension Date {
+    
+    static var ip_millisecondFormatter: DateFormatter {
+        let formatter = DateFormatter()
```

Comment:
Is it intension that some of these methods set the `formatter.locale` and other don't?